test.js

```diff
@@ -184,6 +184,27 @@ test('normalize', function(t) {
     'properties should match attributes (#5)'
   )
 
+  t.equal(
+    normalize(':class'),
+    ':class',
+    `non-word characters should not be removed
+    when they did not serve as a delimiter (#7)`
```

Comment:
Could you use the format `(GH-7, #1)` here and the two below? I use the `#` above as a count.